manifests/0000_90_kube-apiserver-operator_03_servicemonitor.yaml

```diff
@@ -51,7 +51,7 @@ spec:
           Cluster has enabled Technology Preview features that cannot be undone and will prevent upgrades.
           The TechPreviewNoUpgrade feature set is not recommended on production clusters.
       expr: |
-        cluster_feature_set{name!="", namespace="openshift-kube-apiserver-operator"} == 0
+        cluster_feature_set{name=~"TechPreviewNoUpgrade|CustomNoUpgrade", namespace="openshift-kube-apiserver-operator"} == 0
```

Comment:
alright, so any value other than `TechPreviewNoUpgrade` or `CustomNoUpgrade` won't prevent upgrades, correct?

Reply:
Correct, or at the very least it doesn't prevent upgrades because a tech preview feature was enabled. If there comes an alerting use case for a new feature set, we could always append it to the existing alert or create a brand new alert just for it.

Reply:
how will we be able to detect that a new feature has been added that blocks upgrades?

Reply:
it will be hard for us the api team to know, but openshift/api approvers should be aware of that since they would need to approve changes to https://github.com/openshift/api/blob/fd420c26854b53f62bd704b9cb8952a82c281403/config/v1/types_feature.go#L35-L50.

Reply:
@dgrisonnet @deads2k @mfojtik Is the process of adding a new feature documented? Can we rely on it? Maybe we can simply write a basic e2e test that will fail when a new flag blocking upgrades is added? Based on that, we could detect the change and update our alert.

Reply:
No, this will only update alert status. Upgrades will still be blocked by operator's Uprgadable=False status